src/assets/Fonts/Didact_Gothic/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright (c) 2009, 2010, 2011 Daniel Johnson (<il.basso.buffo@gmail.com>).
```

Comment:
'Fonts' folder should start from lowercase

Reply:
I renamed this folder.